### PR TITLE
feat: Adding Documentation for sshnp on Raspberry Pi

### PR DESCRIPTION
**Pull request by TylerTrott**
**- What I did**
Created a step by step guide on installing sshnp on a raspberry pi

**- How I did it**
Local edits and going through each step myself to confirm validity 

**- How to verify it**
walkthrough steps on your own pi :) 

**- Description for the changelog**
Adding Documentation for sshnp on Raspberry Pi

### CODE REVIEW

File: sshnp_raspberrypi_demo/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# SSHNP with a Raspberry Pi Demo
+
+The demonstration features the utilization of a Raspberry Pi 3 Model B. This documentation guides you through the process of setting up a Raspberry Pi from scratch, encompassing the following steps:
+
+### What you will need
+- Raspberry Pi 4 Model B
```
Comment (cpswan) [inline sshnp_raspberrypi_demo/README.md:6]: 3 or 4? It says 3 above.

File: sshnp_raspberrypi_demo/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+- MicroSD card (8GB or more preferably)
+- MicroSD card reader
+- Power supply (5V, 2.5A)
+- Ethernet cable (optional)
+- HDMI to microHDMI cable 
+
+### 1. Downloading the latest version of Raspbian OS 
+You can download the latest version of Raspberry Pi OS (previously called Raspbian) from the official website [here](https://www.raspberrypi.org/downloads/raspbian/). Follow the instructions on the website to install the OS on a microSD card.
```
Comment (cpswan) [inline sshnp_raspberrypi_demo/README.md]: Raspbian (above) is the old name

Comment (cpswan) [reply sshnp_raspberrypi_demo/README.md]: That link now redirects to https://www.raspberrypi.com/software/

**Review by cpswan — CHANGES_REQUESTED** [review]

File: sshnp_raspberrypi_demo/README.md (hunk 1 above)
Comment (TylerTrott) [reply sshnp_raspberrypi_demo/README.md:6]: whoops, Pi 4, good catch

**Review by JeremyTubongbanua — APPROVED** [review]

**Review by JeremyTubongbanua — CHANGES_REQUESTED** [review]
Do you think we could add this demo to the running list of demos in the root `./README.md` with a short description?

image

**Comment by TylerTrott** [comment]
> Do you think we could add this demo to the running list of demos in the root `./README.md` with a short description?
> 
> image

This has been resolved

**Review by JeremyTubongbanua — DISMISSED** [review]

**Review by cpswan — DISMISSED** [review]

**Review by JeremyTubongbanua — APPROVED** [review]

**Review by cpswan — APPROVED** [review]

**Pull request merged**